src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -8411,3 +8422,145 @@ class LambdaExpressionSegment(BaseSegment):
         Ref("FunctionAssignerSegment"),
         Ref("ExpressionSegment"),
     )
+
+
+class PasswordPolicyReferenceSegment(ObjectReferenceSegment):
```

Comment:
as noted with the import
```suggestion
class PasswordPolicyReferenceSegment(ansi.ObjectReferenceSegment):
```